src/libs/antares/checks/checkLoadedInputData.cpp

```diff
@@ -49,6 +51,34 @@ void checkOrtoolsUsage(Antares::Data::UnitCommitmentMode ucMode,
     }
 }
 
+void checkOrtoolsSolverSpecificParameters(
+  const Antares::Data::UnitCommitmentMode& ucMode,
+  const Antares::Solver::Optimization::OptimizationOptions& optOptions)
+{
+    if (optOptions.ortoolsUsed)
+    {
+        MPSolver* solver;
```

Comment:
Maybe not, it seems that if solver specific parameters are not successfully applied, ortools issues a warning and do not crash, so we may not need to check that parameters are set with success, what do you think @flomnes ?

Reply:
Howerver, the user should then carrefully check the logs to see if its parameters are really taken into account

Reply:
It's because Google has a "no crash" policy, but we don't have to have the same policy.
The SetSolverSpecificParameters has a boolean return value. If we want to crash instead of making the user read the logs, I suggest checking this boolean at call time (before solve) and crashing then, instead of creating a disposable MPSolver instance beforehand, which would have a runtime cost

Reply:
Ok I agree with the remark (that was my first idea), but @flomnes mentioned that if we do this way, we do a lot of things before crashing (reading data, constructing the problem), so the user may wait a while before the crash. This is why I tried to do the check earlier.

Reply:
I understand his point of view, but I think it's better to penalize the user "a lot" once (ie in the one study run with wrong parameters, they would have to wait before crash), than to penalize them "a little" each and every time (ie add a small overhead to check that the parameters are correct in every study, after they have corrected the parameters)